docfx_yaml/extension.py

```diff
@@ -626,6 +628,15 @@ def process_docstring(app, _type, name, obj, options, lines):
     """
     This function takes the docstring and indexes it into memory.
     """
+
+    # Check if we already processed this docstring.
+    if name not in app.env.docfx_uid_names:
```

Comment:
I would flip this, returning early if it's already processed. Then, you don't need the `else`.

Reply:
done!